web/src/app/util/OtherActions.js

```diff
@@ -16,71 +16,70 @@ const cancelable = (_fn) => {
   return cFn
 }
 
-export default class OtherActions extends React.PureComponent {
-  static propTypes = {
-    actions: p.arrayOf(
-      p.shape({
-        label: p.string.isRequired,
-        onClick: p.func.isRequired,
-      }),
-    ).isRequired,
-    color: p.string,
-    icon: p.element,
-    placement: p.oneOf(['left', 'right']),
-  }
+export default function OtherActions(props) {
+  
+  const [state, setState] = useState({
```

Comment:
Rather than state/setState, lets do
```suggestion
  const [anchorEl, setAnchorEl] = useState(null)
```

that way we can use `anchorEl` directly without having to reference an object